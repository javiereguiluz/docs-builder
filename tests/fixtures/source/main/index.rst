Some Test Docs!
===============

.. image:: /_images/symfony-logo.png

.. toctree::

    datetime
    form/form_type

.. _reference-forms-type-date-format:

A header with ``a code literal``
--------------------------------

Some info...
