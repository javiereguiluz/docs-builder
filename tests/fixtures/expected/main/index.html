<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="section" id="some-test-docs">
    <h1>Some Test Docs!<a class="headerlink" href="#some-test-docs" title="Permalink to this headline">¶</a></h1>
    <img src="_images/symfony-logo.png" />
    <div class="toctree-wrapper compound">
        <ul>
            <li><a href="datetime.html#datetimetype-field">DateTimeType Field</a>
                <ul>
                    <li><a href="datetime.html#field-options">Field Options</a></li>
                    <li><a href="datetime.html#overridden-options">Overridden Options</a></li>
                    <li><a href="datetime.html#field-variables">Field Variables</a></li>
                    <li><a href="datetime.html#url-checker-errors">Url checker errors</a></li>
                </ul>
            </li>
            <li><a href="form/form_type.html#formtype-documentation">FormType Documentation</a></li>
        </ul>
    </div>
<span id="reference-forms-type-date-format"></span>
<div class="section" id="a-header">
    <h2>A header with <code>a code literal</code><a class="headerlink" href="#a-header" title="Permalink to this headline">¶</a></h2>
    <p>Some info...</p>
</div>
</div>
</body>
</html>
